app/dashboard/migrations/0151_auto_20200921_1510.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 2.2.4 on 2020-09-21 15:10
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('dashboard', '0150_auto_20200913_1437'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='hackathonproject',
+            name='grant_link',
+            field=models.ForeignKey(help_text='Link to grant if project is converted to grant', null=True, on_delete=django.db.models.deletion.SET_NULL, to='grants.Grant'),
+        ),
+    ]
```

Comment:
^ squash migrations

Reply:
I squashed the migrations, this file is from the older commits won't be included. https://github.com/gitcoinco/web/blob/feature/convert2grant/app/dashboard/migrations/0149_hackathonproject_grant_link_squashed_0151_auto_20200921_1510.py

Reply:
I removing the last files to fix this.